common/actions.js

```diff
@@ -98,7 +98,15 @@ export const getSlateById = async (data) => {
 
 export const getSlatesByIds = async (data) => {
   await Websockets.checkWebsocket();
-  return await returnJSON(`/api/slates/get`, {
+  return await returnJSON(`/api/v1/explore`, {
```

Comment:
Are you sure nothing else is using `getSlatesByIds`?

Reply:
Its okay to change the internals of a function if nothing else is using it. But then this function is named incorrectly then?

Reply:
So currently nothing is using this endpoint, but generally it's not a good idea to changes things like this. I don't think you are using this either tara so maybe change it back so that its behavior is predictable. 

FYI when you call Data.getSlatesById, it's not the same as this (which would be called with Actions.getSlatesById). Data.getSlatesById calls node_common/data/methods/get-slates-by-ids. Data is on the backend, Actions is on the front end

Reply:
yep I know, I was just thinking that we could probably keep the endpoint just in case. 
I will change it back